eventtracking/backends/segment.py

```diff
@@ -0,0 +1,64 @@
+"""Event tracking backend that sends events to segment.com"""
+
+from __future__ import absolute_import
+
+try:
+    import analytics
+except ImportError:
+    analytics = None
+
+
+class SegmentBackend(object):
+    """
+
+    Send events to segment.com
+
+    It is assumed that other code elsewhere initializes the segment.com API and makes calls to analytics.identify.
```

Comment:
Yes - the library has a pattern for initializing itself in a Django environment that ensures it is only initialized once (which we do in the LMS). We also have implemented the login hook that calls analytics.identify when the user successfully authenticates.